modules/auth/models/user.php

```diff
@@ -109,29 +109,13 @@ public function __isset($property) {
 	}
 
 	/**
-	 * Retrieves the users avatar, currently this attempts to retrieve
-	 * it from gravatar.
+	 * Returns the url for the avatar
 	 *
-	 * @param $size The size of the avatar image in pixels
 	 * @return string The URL to access the avatar
 	 */
-	public function get_avatar_url ($size = 28) {
-
```

Comment:
Not sure if any other repo is using this function, so I chose to keep it and just return the url to the icon.

Reply:
A quick `grep` in our repos seems to report no other usage. I think it's safe to remove it!